wp-content/plugins/vf-masthead-container/group_vf_masthead.json

```diff
@@ -110,7 +110,7 @@
             "label": "Background image",
             "name": "vf_masthead_image",
             "type": "image",
-            "instructions": "Masthead dimensions: 2448px by 300px",
+            "instructions": "In choosing the image for masthead, the image filename needs to have it's most dominate colour added as a hexadecimal code. ***masthead-background-image--00EF00.png*** To find the dominant colour in the background image you can use a colour picker in graphics software to find the hex code.",
```

Comment:
This might be something we could ask David for help with. So if a user doesn't have a hex code, they could specify it in the UI. Or, perhaps, a simple toggle if text should be white or black. 

But, in any case, something to be done "later"

@sturobson

Reply:
Yes, that may be quite confusing for the users. Definitely, something to think of.

Reply:
This is what I was thinking when my internet failed at standup …

perhaps a tool where

1. the image is uploaded.
2. the user gets a preview with a color picker.
3. the color picked gets added to the end of the image.


Or - we could get JS to do even more -- http://jsfiddle.net/xLF38/818/ 

perhaps that might be better - although there would probably be a 'flash of style'

Reply:
actually -- that's probably a nicer solution -- and something for me to try this afternoon.

Reply:
hmmm, a quick play with that fiddle and it has the embl.org first hero as 'mainly black' so it's black - that won't work so good.

Back the idea of having a 'step' between upload and publish with a colour picker